### PR TITLE
feat(WizardModal): add WizardModal

### PR DESCRIPTION
**Pull request by Mnau-mnau**
implements https://zitenote.atlassian.net/browse/UXD-630

Wizard modal follows the sizing of the regular modal, but it doesn't have the self-closing functionality. This is upon the component consumer. It also blurs the background.

There is no enforced padding so far - the lgPlus looks weird on sizes xs and small - feel free to comment

### CODE REVIEW

File: src/components/WizardModal/WizardModal.docs.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+<Meta title="components/modals/SemanticModal/Docs" />
+
+# Modal documentation
+
+>**[Confluence page](https://zitenote.atlassian.net/l/c/ur1JbSHg)** about pattern usage, rationale, planned changes, and any current discrepancies between Storybook and Figma.
+
+> **Visual documentation**: please open in Figma for full context, these are only the most prominent examples of the pattern.
+
+<Figma url="https://www.figma.com/file/i13gqrwTYjmMhHkEcwaM3x/Persona-based-Onboarding?node-id=279%3A8312"/>
```
Comment (ajkl2533) [inline src/components/WizardModal/WizardModal.docs.mdx:12]: I would replace this link with https://www.figma.com/file/vnzYfrwIISRYkr2iYE5TcD/DS-Atoms?node-id=24200%3A24909

File: src/components/WizardModal/WizardModal.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { ModalSizes } from '../Modal/Modal.enums';
+import { WizardModalProps } from './WizardModal.types';
+import { Padbox } from '../layout';
+
+const widthVariants = {
+  [ModalSizes.xs]: 320,
+  [ModalSizes.sm]: 430,
+  [ModalSizes.md]: 600,
+  [ModalSizes.lg]: 900,
+};
```
Comment (ajkl2533) [inline src/components/WizardModal/WizardModal.tsx:16]: I personally think that this modal definitely should not exist in `xs` and `sm` sizes, we can discuss the `md` (but I would remove that also). 
Please follow Figma discussion

File: src/components/WizardModal/WizardModal.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  border-radius: ${getRadii('default')};
+  background-color: ${getColor('neutral.0')};
+  overflow: auto;
+  > * {
+    height: 100%;
+  }
+`;
+
+const WizardModal: React.FC<WizardModalProps> = ({ children, size = 'lg' }) => {
+  const { Portal } = usePortal();
```
Comment (ajkl2533) [inline src/components/WizardModal/WizardModal.tsx:44]: Please import `usePortal` from 'src/hooks/usePortal` that hook has already defined some default values.

File: src/components/WizardModal/WizardModal.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
```
Comment (ajkl2533) [inline src/components/WizardModal/WizardModal.tsx:45]: please add `useLockBodyScroll()` to prevent body scrolling when modal is active

**Review by ajkl2533 — APPROVED** [review]
LGTM, couple of comment and we need to resolve sizes with UX

File: src/components/WizardModal/WizardModal.tsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+import { useLockBodyScroll } from '../../hooks/useLockBodyScroll';
+import { getColor, getDepth, getRadii, pxToRem } from '../../utils';
+import { WizardModalSizes } from './WizardModal.enums';
+import { WizardModalProps } from './WizardModal.types';
+import { Padbox } from '../layout';
+
+const widthVariants = {
+  [WizardModalSizes.md]: 600,
+  [WizardModalSizes.lg]: 900,
+};
```
Comment (ajkl2533) [inline src/components/WizardModal/WizardModal.tsx:15]: We can completely drop the `size` property. This modal will exist only in `lg` variant

Comment (Mnau-mnau) [reply src/components/WizardModal/WizardModal.tsx]: dropped

**Pull request merged**